mqtt.kafka.asyncapi.proxy/docker/compose/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  zilla:
+    image: ghcr.io/aklivity/zilla
+    container_name: zilla
+    pull_policy: always
+    restart: unless-stopped
+    ports:
+      - 7183:7183
+    environment:
+      KEYSTORE_PASSWORD: generated
+      KAFKA_HOST: ${KAFKA_HOST}
+      KAFKA_PORT: ${KAFKA_PORT}
+      JAVA_OPTIONS: -Dzilla.incubator.enabled=true
+    volumes:
+      - ../../zilla.yaml:/etc/zilla/zilla.yaml
+      - ../../mqtt/asyncapi.yaml:/etc/zilla/mqtt/mqtt-asyncapi.yaml
```

Comment:
the [asyncapi examples](https://github.com/asyncapi/spec/tree/master/examples) have a file name based organization instead of a folder based one. I think it will be better to just name these `mqtt-asyncapi.yaml` and `kafka-asyncapi.yaml` respectively and remove the folders since they are redundant. It will also simplify the zilla.yaml